database/languages/python.json

```
@@ -26,5 +26,12 @@
         "url": "https://www.learnpython.org/",
         "category": "languages",
         "subcategory": "python"
+    },
+    {
+        "name": "Real Python Tutorial",
+        "description": "Learn Python through Real Hands on experience",
```

[Nitpick] This description could use more information. What other ways can developers and other tech professionals can benefit from this resource?
```suggestion
        "description": "Learn Python through Real Hands on experience",
```

---

Hi @CBID2 , I have updated the description. Please have a look at it and let me know if any other changes are required:)

---

Hi @preethi1915. There are some merge conflicts in your branch, so work on that. :)